src/extensions/default/InlineColorEditor/unittests.js

```diff
@@ -256,25 +266,23 @@ define(function (require, exports, module) {
                     });
                 });
                 
-                it("should not update the end bookmark to a shorter valid match if the bookmark still exists and the color becomes invalid", function () {
+                it("should not update the end bookmark and the color shown to a shorter valid match if the bookmark still exists and the color becomes invalid", function () {
```

Comment:
All of the references to "bookmark" should be updated to something like "textmark".